File: src/main/java/org/jukeboxmc/potion/ConduitPowerEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ConduitPowerEffect extends Effect {

    @Override
    public int getId() {
        return 26;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.CONDUIT_POWER;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 29, 194, 209 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/ConduitPowerEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class ConduitPowerEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 26
    }

    override fun getEffectType(): EffectType {
        return EffectType.CONDUIT_POWER
    }

    override fun getEffectColor(): Color {
        return Color(29, 194, 209)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}